### ISSUE TITLE
Stuck: Generating login keys

### ISSUE THREAD

**Issue by imacfan** [standardnotes/desktop#742]
I have been trying to login on macOS ..
Entered email, password and sync server domain.

But it is always stuck at "Generating login keys". I have tried multiple versions. All of them give same issue.
Mobile app and web  works fine with same account and sync server.

What could be the issue?

**Comment by JaspalSuri** [comment]
Hi @imacfan, since you've mentioned entering a syncing server domain:

- Are you using the old setup that's primarily based on Ruby and Ruby on Rails or the new setup that's primarily based on JavaScript? 
- If you're using the latter, are you using [SSJS](https://github.com/standardnotes/syncing-server-js/) or [Standalone](https://github.com/standardnotes/standalone)?

**Comment by imacfan** [comment]
Yeah possibly, I am using the server that comes via Yunohost. Need to check if it's the latest sever setup.
